tests/functional/includes/date.mock.js

```diff
@@ -1,14 +1,30 @@
-// Highjack the Date object so that our tests don't suddenly start failing...
+(function () {
+    'use strict';
 
-var __Date__ = Date;
+    // Create a date object for a specific instant in time (my 36th birthday...)
 
-// February 13, 2015
-var STATIC_DATE_MS = 1423852666565;
+    var d = new Date();
+    d.setUTCFullYear(2014, 11, 8);
+    d.setUTCHours(12, 34, 0, 0);
 
-Date = function (value) {
-    return new __Date__(value !== undefined ? value : STATIC_DATE_MS);
-};
+    // Adjust it based on the time zone offset, so that formatted date/time
+    // will be what we expect, independent of the time zone (minus the time
+    // zone name, which is a different story)
 
-Date.now = function () {
-    return STATIC_DATE_MS;
-};
+    d.setUTCMinutes(d.getUTCMinutes() + d.getTimezoneOffset());
```

Comment:
It means that if you specify 90, for example, it will modify both hours and minutes (i.e. it will add 1 hour, and set the minutes to 30). I remember using date objects extensively as far back as 2003 (was working on a web-based calendar at the time) and the behavior has not changed since. It just isn't well documented.